gapis/api/vulkan/api/copy_clear_commands.api

```diff
@@ -627,7 +627,7 @@ sub void copyImageBuffer(VkBuffer buffer, VkImage image, VkImageLayout layout, m
     aspectBit := aspectBits[0]
```

Comment:
Nit: there should be one more level of indentation in the block starting here, as we're in the `else` part of the `if` in the line above. See the `}}` near the end of this `sub`.